README.md

```diff
@@ -159,6 +159,9 @@ The Management API host
 #### `proxy` [string]
 Proxy configuration in HTTP auth format: `host:port` or `user:password@host:port`
 
+#### `rawProxy` [boolean]
```

Comment:
Same comment as the import tool, can we add some info about this in the troubleshooting section